2023-05-25 15:53:18,599 ERROR 90398 [-/127.0.0.1/-/0ms GET /] nodejs.ER_ACCESS_DENIED_ERRORError: ER_ACCESS_DENIED_ERROR: Access denied for user 'root'@'localhost' (using password: NO)
    at Sequence._packetToError (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
    --------------------
    at Protocol._enqueue (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/Pool.js:48:16)
    at node:internal/util:367:7
    at new Promise (<anonymous>)
    at Pool.getConnection (node:internal/util:353:12)
    at Pool.query (/Users/stone_x/Desktop/Personal Ledger/backend/node_modules/mysql/lib/Pool.js:202:8)
    at node:internal/util:367:7
    at new Promise (<anonymous>)
    sql: select now() as currentTime;
code: "ER_ACCESS_DENIED_ERROR"
errno: 1045
sqlMessage: "Access denied for user 'root'@'localhost' (using password: NO)"
sqlState: "28000"
fatal: true
name: "ER_ACCESS_DENIED_ERRORError"
pid: 90398
hostname: Stone-xdeMacBook-Pro.local

